wpilib/wpilib/timedrobot.py

```diff
@@ -0,0 +1,34 @@
+# validated: 2017-11-09 TW ef3267833fc3 edu/wpi/first/wpilibj/TimedRobot.java
```

Comment:
I'd like to pull in the IterativeRobotBase changes ASAP. I say TimedRobot can wait for another PR.

Reply:
It'll have to wait anyway for the notifier

Reply:
Yeah, but this PR is now holding up actually finishing the LiveWindow changes.

Reply:
Yeah I agree. Now I gotta figure out the best way to separate IterativeRobotBase...